news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: 4 Digit World Cup 2024 Concludes
+date: 2024-06-24 12:00:00 +0000
+---
+
+This year's 4 Digit World Cup has finally come to an end! Join us for a recap on everything that's been going on as well as some insights from one of our second-place finishers, a commentator and a custom mapper.
+
+![](/wiki/shared/news/2024-03-21-4-digit-world-cup-2024-registrations-now-open/banner@2x.jpg)
+
+Over the past two months, the 4-digit tournament scene has showcased their best players competing at the highest level within the rank range. Now, with the Grand Finals weekend behind us, it's time to reflect on the intense matches, unforgettable moments, and the friends we made along the way.
+
+This year's 4WC has seen many changes compared to the previous iteration to stay up to date in the everchanging nature of osu! tournaments. This includes a more consistent difficulty scaling, an increase in custom beatmaps, a larger involvement with the playerbase as well as taking in new and talented staff members to help them flourish and elevate the 4 Digit World Cup to newer heights with their sparkling passion and creativity!
+
+## Summary
+
+![](/wiki/Tournaments/4WC/2024/img/podium.jpg)
+
+Congratulations to the United States for making 4WC history by being the first claim the title of champions twice in a row, as well as being the first team to garner three total 4WC wins. The United States had their strongest showing to date in this iteration of the tournament, only losing 10 points total in their 6 matches. Canada put in a strong effort, but there was never a moment where the United States was on the backfoot, and with half of their team being new to the roster, their depth remains unmatched.
+
+As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future. China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers' bracket, winning 7 matches in a row to claim their #2 slot once more.
```

Comment:
```suggestion
As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future.

China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers' bracket, winning 7 matches in a row to claim their #2 slot once more.
```